internal/pkg/cache/cache.go

```diff
@@ -15,8 +15,8 @@ const (
 	DefaultRegistryName   = "default"
 	DefaultRegistrySource = "github.com/hashicorp/nomad-pack-community-registry"
 	DefaultRef            = "latest"
-	DevRegistryName       = "dev"
-	DevRef                = "dev"
+	DevRegistryName       = "<<local folder>>"
```

Comment:
I like this improvement a lot. This was super last minute addition and didn't get thought through as well as it should have.